spark-indexer/src/test/java/org/uniprot/store/spark/indexer/uniprot/UniProtKBDocumentsToHDFSWriterTest.java

```diff
@@ -20,9 +12,16 @@
 import org.uniprot.store.spark.indexer.common.JobParameter;
 import org.uniprot.store.spark.indexer.common.util.SparkUtils;
 import org.uniprot.store.spark.indexer.uniprot.mapper.UniProtEntryToSolrDocument;
-
 import scala.Tuple2;
 
+import java.util.*;
```

Comment:
Here, I think, you formatted the class with InteliJ. Can you please run the maven command to make sure it is using maven format ?